include/evmc/evmc.hpp

```diff
@@ -15,6 +15,8 @@
 /// @ingroup cpp
 namespace evmc
 {
+#define B(V, INDEX) static_cast<uint8_t>((V) >> (8 * (INDEX)))
```

Comment:
why not a constexpr function?

Reply:
The name must be short to get formatting reasonable. And I can undef it later.

Reply:
I'm not a fan of macros with really short names. Also to me the original long formatted version with `static_cast`s doesn't look that bad to warrant a bad macro :)

Reply:
Well if someone has another `#define B` before `#include <evmc/evmc.hpp>`, this will break it.

You could maybe group lines by 4-5 bytes to make formatting work with longer name.


Also lint complains about missing documentation for it.